README.md

```diff
@@ -14,6 +14,7 @@ System and Service Manager
 [![Fossies codespell report](https://fossies.org/linux/test/systemd-main.tar.gz/codespell.svg)](https://fossies.org/linux/test/systemd-main.tar.gz/codespell.html)</br>
 [![Coverage Status](https://coveralls.io/repos/github/systemd/systemd/badge.svg?branch=main)](https://coveralls.io/github/systemd/systemd?branch=main)</br>
 [![Packaging status](https://repology.org/badge/tiny-repos/systemd.svg)](https://repology.org/project/systemd/versions)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/systemd/systemd/badge)](https://api.securityscorecards.dev/projects/github.com/systemd/systemd)
```

Comment:
Maybe one more question - should the badge point to the JSON endpoint? Or is there some "human-friendly" dashboard with all the alerts this should point to instead?

Reply:
The Scorecard team is working on improving this interface to be more "human-friendly"